repo.go

```diff
@@ -695,6 +696,41 @@ func (r *Repo) AddTargets(paths []string, custom json.RawMessage) error {
 	return r.AddTargetsWithExpires(paths, custom, data.DefaultExpires("targets"))
 }
 
+func (r *Repo) AddDigestTargets(digest string, length int64, custom json.RawMessage, path string) error {
+	expires := data.DefaultExpires("targets")
+
+	t, err := r.targets()
```

Comment:
For a future PR:
```suggestion
	// TODO: support delegated targets
	t, err := r.targets()
```

Reply:
Yes, this depends on the order of this and #175, but would be good to support